sandy/errorr.py

```diff
@@ -91,17 +91,18 @@ def get_cov(self):
         Examples
         --------
         >>> endf6 = sandy.get_endf6_file("jeff_33", "xs", 10010)
-        >>> err = endf6.get_errorr(ek=sandy.energy_grids.CASMO12, err=1)
-        >>> err.get_cov().data.iloc[:5, :5]
-                MAT1	125
-                MT1	1
-                E1	1.00000e-05	3.00000e-02	5.80000e-02	1.40000e-01	2.80000e-01
-        MAT	MT	E					
-        125	1	1.00000e-05	8.82899e-06	8.65234e-06	8.60908e-06	8.58043e-06	8.57036e-06
-                3.00000e-02	8.65234e-06	8.65909e-06	8.66074e-06	8.66183e-06	8.66221e-06
-                5.80000e-02	8.60908e-06	8.66074e-06	8.67339e-06	8.68176e-06	8.68471e-06
-                1.40000e-01	8.58043e-06	8.66183e-06	8.68176e-06	8.69497e-06	8.69961e-06
-                2.80000e-01	8.57036e-06	8.66221e-06	8.68471e-06	8.69961e-06	8.70484e-06
+        >>> err = endf6.get_errorr(ek=[1e-2, 1e3], err=1)
```

Comment:
you are not giving two groups, only one, defined by two boundaries.
You can define 2 groups with, for example, `ek=[1e-5, 1e1, 2e7]`